components/Python/RestModelServing/restful_sklearn_serving/sklearn_restful_serving.py

```diff
@@ -218,7 +218,8 @@ def predict(self, url_params, form_params):
                                        .format(pred_probs,  pred_index, prediction, prediction_confidence))
 
                     # Total confidence
-                    self._total_confidence_metric.increase(prediction_confidence)
+                    if self._total_confidence_metric:
```

Comment:
Following a discussion with Yakov, we've agreed that trying to run the restful components by using the `__name__ == __main__` should not be supported anymore. It leads to potential weird and complicated bugs. Currently we support a standalone mode via `mlpiper` command line tool.